app/controllers/concerns/encourage_second_mfa_concern.rb

```diff
@@ -0,0 +1,26 @@
+module EncourageSecondMfaConcern
+  def user_needs_second_mfa_encouragement?
+    return false if user_has_dismissed_second_mfa_encouragement? || user_has_multiple_mfa_methods?
+    user_meets_sign_in_count_for_encouragement? || user_meets_time_elapsed_for_encouragement?
+  end
+
+  private
+
+  def user_has_dismissed_second_mfa_encouragement?
```

Comment:
Do we only want to Encourage once? as in if the user dismisses it they should never be encouraged again? or is there any point where we will want to re-encourage after the first dismissal.

Reply:
> Do we only want to Encourage once? as in if the user dismisses it they should never be encouraged again? or is there any point where we will want to re-encourage after the first dismissal.

My read of the ticket is that this should only happen once, yes. There's a short blurb in the ticket "For now, this will only occur once"